tests/bluetooth/bsim_bt/bsim_test_mesh/src/test_persistence.c

```diff
@@ -42,7 +44,9 @@ static int test_ividx = 0x123456;
 static uint8_t test_flags;
 static uint8_t test_netkey_idx = 0x77;
 static uint8_t test_netkey[16] = { 0xaa };
+static struct bt_mesh_key mesh_test_netkey;
```

Comment:
Nit-picking (just to align with other code):
```suggestion
static struct bt_mesh_key mesh_test_net_key;
```
There are few more cases where `netkey` or `devkey` are used without underscore.